app/shop.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from app.convert import shop_list
+
+
+@dataclass
+class Shop:
+    name: str
+    location: []
+    product_list: {}
+
+    def pack_the_bucket(self, wish_list: dict) -> float:
+        return (
+            sum([wish_list[article] * self.product_list[article]
+                for article in wish_list])

```

Comment:
not need to create list
```suggestion
        return (
            sum(wish_list[article] * self.product_list[article]
                for article in wish_list)

```